### PR TITLE
Tweak homepage content

### PR DESCRIPTION
**Pull request by nottrobin**
- Add links to "project" names
- Make sure external link icons look okay
- Change documentation link text and targets

### CODE REVIEW

File: templates/index.html [hunk 1]
```diff
@@ -17,53 +17,28 @@
...
-                </td>
-                <td>
-                    <p>
-                        Documentation for MAAS (Metal as a Service).
-                    </p>
-                    <p>
-                        <a href="/maas">Learn more about MAAS &gt;</a>
-                    </p>
+                    <p>Documentation for MAAS (Metal as a Service).</p>
+                    <p><a href="/maas">Read the MAAS documentation &gt;</a></p>
```
Comment (anthonydillon) [inline templates/index.html]: Please use `&nbsp;&rsaquo;`

File: templates/index.html [hunk 2]
```diff
@@ -17,53 +17,28 @@
...
-                </td>
-                <td>
-                    <p>
-                        Documentation for the next generation OS from Ubuntu for all your IoT devices.
-                    </p>
-                    <p>
-                        <a href="/core">Learn more about Ubuntu Core &gt;</a>
-                    </p>
+                    <p>Documentation for the next generation OS from Ubuntu for all your IoT devices.</p>
+                    <p><a href="/core">Read the Ubuntu Core documentation &gt;</a></p>
```
Comment (anthonydillon) [inline templates/index.html]: Same here

File: templates/index.html [hunk 3]
```diff
@@ -73,7 +48,10 @@
 <div class="row">
     <h2>What is this site?</h2>
     <p>
-        The Ubuntu documentation site is a service that makes it easy to publish product documentation from github or Launchpad that has been written in markdown format with a defined table of contents and in a simple directory structure.  It supports embedding images and versions of documents.  If you are interested in moving your documentation to this service, please get in touch.
+        The Ubuntu documentation site is a service that makes it easy to publish product documentation from github or Launchpad
+        that has been written in markdown format with a defined table of contents and in a simple directory structure.
+        It supports embedding images and versions of documents.
+        If you are interested in moving your documentation to this service, please <a href="mailto:webteam@lists.canonical.com">get in touch</a>.
```
Comment (anthonydillon) [inline templates/index.html]: This should be a link to join the mailing list instead of the mailto.

**Comment by nottrobin** [comment]
@anthonydillon addressed - please approve.

**Pull request merged**